Volt/Volt/src/Volt/Scripting/Mono/MonoScriptGlue.h

```diff
@@ -1,4 +1,5 @@
 #pragma once
+#pragma once
```

Comment:
Remove the duplicate pragma once.

Reply:
Fixed in commit [9d6db14](https://github.com/ChunkTreasure1/Volt/pull/37/commits/9d6db144debf099bb2672b717cfa96364ca45450)